src/showingpreviously/cinemas/savoy.py

```diff
@@ -0,0 +1,130 @@
+import json
+import re
+from datetime import datetime, timedelta
+from bs4 import BeautifulSoup
+
+import showingpreviously.requests as requests
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+from showingpreviously.consts import UK_TIMEZONE, UNKNOWN_FILM_YEAR, STANDARD_DAYS_AHEAD
+
+
+CINEMAS_URL = 'http://www.savoycinemas.co.uk/'
+CHAIN = Chain('Savoy Cinemas')
+EVENTS_INLINE_PATTERN = re.compile(r'var\s+eventsInline\s*=\s*(?P<events_inline>(?:.|\n|\r)*?]);')
+
+
+def get_response(url: str) -> requests.Response:
+    r = requests.get(url)
+    if r.status_code != 200:
+        raise CinemaArchiverException(f'Got status code {r.status_code} when fetching URL {url}')
+    return r
+
+
+def parse_film_name(name: str) -> (str, [str]):
+    attributes = {'format': []}
+    # ending
+    for removal in [' U', '(PG)', '(12A)', '(12)', '(15)', '(18)', '2D', '3D', '4K', '(WITH SANTA)']:
+        if name.lower().endswith(removal.lower()):
+            name = name[:-1 * len(removal)].strip()
+            if removal in ['2D', '3D', '4K']:
+                attributes['format'].append(removal)
+    # starting
+    for removal in ['NATIONAL THEATRE LIVE:']:
+        if name.lower().startswith(removal.lower()):
+            name = name[len(removal):].strip()
+            if removal == 'NATIONAL THEATRE LIVE:':
+                attributes['format'].append('live')
+                attributes['event'] = True
+    if len(attributes['format']) == 0:
+        del attributes['format']
+    return name, attributes
+
+
+def get_cinemas_as_dict() -> dict[str, Cinema]:
+    r = get_response(CINEMAS_URL)
+    soup = BeautifulSoup(r.text, features='html.parser')
+    cinemas = {}
+    for cinema_link in soup.find_all('a', {'href': True}):
+        cinema_url = cinema_link['href']
+        cinema_name = cinema_link.find('div', {'class': 'title'}).text
+        cinemas[cinema_url] = Cinema(cinema_name, UK_TIMEZONE)
+    return cinemas
+
+
+def get_attributes(showing: dict[str, any], attributes) -> dict[str, any]:
+    if showing['PB'] != 'N':
+        attributes['carers-and-babies'] = True
+    if showing['SS'] != 'N':
+        attributes['senior'] = True
+    return attributes
+
+
+def get_screen_inline(booking_url: str) -> Screen:
+    r = requests.get(booking_url)
```

Comment:
Why are we using `requests.get()` directly here and not the `get_response()` method?